LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2017 oprogramador
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
Sorry for a delay but it seems that this text differs from https://opensource.org/licenses/MIT .

I didn't have enough time to find out the exact differences.

Reply:
Yes, I'm aware of the difference. It is the addition of the one line at the top: "MIT License". The text at https://opensource.org/licenses/MIT doesn't have it. The version with the additional line was provided by GitHub. It's their standard template for the MIT License.

Reply:
I see also a different division into lines.